src/orquestra/sdk/_base/_workflow.py

```diff
@@ -31,6 +31,7 @@
 
 from .. import secrets
 from . import _api, _dsl, loader
+from ._api._config import _resolve_config
```

Comment:
**nitpick**: we shouldn't import private functions across modules. Please remove the leading `_`.